bedrock/mozorg/tests/test_helper_misc.py

```diff
@@ -24,15 +24,15 @@
 
 TEST_DONATE_LOCALE_LINK = {
     'default':
-        'https://sendto.mozilla.org/page/contribute/Give-Now?source={source}',
+        'https://donate.mozilla.org/en-US/?source={source}',
     'en-US':
-        'https://sendto.mozilla.org/page/contribute/givenow-seq?'
-        'preset=2&source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content={source}',
+        'https://donate.mozilla.org/?amount=50'
```

Comment:
Yeah, I think I agree. If you did something like that, we could get Andrea to give us a list of slightly different amounts and presets for mozilla.org, which might perform better.

I would also suggest to set the default amount to be the same as the second (middle button) in the preset.